### ISSUE TITLE
Fix optimisation

### ISSUE THREAD

**Issue by JamesCarr1** [JamesCarr1/Connect2_AI#2]
For some reason, the prior predictions are not working very well and change far less than the value predictions. Try to fix this.

**Comment by JamesCarr1** [comment]
Firstly, the loss function (CrossEntropyLoss) required LOGITS, not predictions=softmax(logits). Therefore adjusted that and performance is better.

**Comment by JamesCarr1** [comment]
Tried changing all softmaxes to log_softmaxes (since that is the assumed activation fn for cross entropy loss), but the loss becomes VERY negative.
